vividus-plugin-rest-api/src/main/java/org/vividus/http/PublishingAttachmentInterceptor.java

```diff
@@ -80,25 +80,28 @@ public void process(HttpRequest request, EntityDetails entityDetails, HttpContex
                 }
             }
         }
-        attachApiMessage("Request: " + request, request.getHeaders(), body, mimeType, -1);
+        attachApiMessage("Request: " + request, request.getHeaders(), body, mimeType, -1, 0);
     }
 
     @Override
-    public void handle(HttpResponse response) throws IOException
+    public void handle(HttpResponse response)
     {
         Header[] headers = response.getResponseHeaders();
         String attachmentTitle = String.format("Response: %s %s", response.getMethod(), response.getFrom());
         String mimeType = getMimeType(headers).orElseGet(ContentType.DEFAULT_TEXT::getMimeType);
-        attachApiMessage(attachmentTitle, headers, response.getResponseBody(), mimeType, response.getStatusCode());
+        attachApiMessage(attachmentTitle, headers, response.getResponseBody(), mimeType, response.getStatusCode(),
+            response.getResponseTimeInMs());
```

Comment:
6 arg to much ? 
can be replaced with `response`

Reply:
you won't be able to use `response` in `attachApiMessage` since it's used to requests as well